thirdparty/README.md

```diff
@@ -75,6 +75,16 @@ Files extracted from upstream source:
 - Version: git (3aaca635bad074a0ce5c15fa8aa0dff47f5c639a, 2023)
 - License: MPL 2.0
 
+## clipper2
+
+- Upstream: https://github.com/AngusJohnson/Clipper2
+- Version: 1.2.2  (756c5079aacab5837e812a143c59dc48a09f22e7, 2023)
+- License: Boost Software License 1.0
+
+Files extracted from upstream source:
+
+- `CPP/Clipper2Lib` folder
+- `LICENSE`
```

Comment:
I don't see the license?

Reply:
Also needs to be documented like this in `COPYRIGHT.txt`:
```diff
diff --git a/COPYRIGHT.txt b/COPYRIGHT.txt
index 3200eafa00..ce9be3dd5e 100644
--- a/COPYRIGHT.txt
+++ b/COPYRIGHT.txt
@@ -171,6 +171,11 @@ Comment: CA certificates
 Copyright: Mozilla Contributors
 License: MPL-2.0
 
+Files: ./thirdparty/clipper2/
+Comment: Clipper2
+Copyright: 2010-2013, Angus Johnson
+License: BSL-1.0
+
 Files: ./thirdparty/cvtt/
 Comment: Convection Texture Tools Stand-Alone Kernels
 Copyright: 2018, Eric Lasota
```

Reply:
Seems I evaporated that file on a rebase by accident.